### DISCUSSION TITLE
Disable copy on single letters delete in Hamsket

### DISCUSSION THREAD

**Discussion by francwalter** [hluk/CopyQ#2298] [Q&A] [answered]
Hi
in Hamsket (a multimessenger) I have a strange phenomenon:
I write some text in the chat box and delete single letters with Backspace and CopyQ copies every single letter.
It looks funny, here i wrote abcdefghijklmno and deleted it slowly:

![image](https://user-images.githubusercontent.com/927269/224726947-07ef8519-2746-4470-8498-44550c2a4a97.png)

If I delete faster, only some of the letters are copied by CopyQ.
In other text editions I couldnt notice the same thing, only in Hamsket (clone of Rambox).

Is there something I can do to ommit that?

Thank.frank

CopyQ 6.4.0 on Ubuntu 22.04

**Comment by hluk** [comment, marked as answer]
This can be done using "Ignore items with no or single character" predefined command. You can add it from Command dialog (F6 shortcut) clicking on Add button. The command avoids storing the characters in CopyQ.

This happens also in web version of WhatsApp. Very annoying.

**Reply by anasaboayham** [reply]
CopyQ

**Comment by francwalter** [comment]
Thank, will do it 😄 
CopyQ is very versatile 👍
